config/routes.rb

```diff
@@ -2,4 +2,5 @@
 
 FlowcommerceSpree::Engine.routes.draw do
   post '/event-target', to: 'webhooks#handle_flow_web_hook_event'
+  get '/stock', to: 'inventory#stock'
```

Comment:
Could we use a more descriptive endpoint? If flow only allows GET method let's use smt like GET /online-stock-availability

Reply:
Updated!! I like more your naming. makes more sense as well!! Thanks